packages/Breadcrumbs/Item/Item.jsx

```diff
@@ -66,13 +66,12 @@ Item.displayName = 'Item'
 
 Item.propTypes = {
   /**
-   * Target URL.
+   * Target URL. This will be converted to `to` if the `reactRouterLinkComponent`
+   * prop is provided to the `Item` or parent `Breadcrumbs` element.
    */
   href: PropTypes.string.isRequired,
   /**
-   * @ignore
-   *
-   * React Router Link component. The reactRouterLinkComponent property will be passed down from from the parent `<Breadcrumbs>`.
+   * React Router Link component
```

Comment:
Can we keep the existing description and add the fact that it can be overwritten.

> React Router Link component. The `reactRouterLinkComponent` property will be passed down from the parent `<Breadcrumbs>` if the parent has a `reactRouterLinkComponent` provided.